packages/docs/src/routes/docs/components/context/index.mdx

```diff
@@ -53,7 +57,7 @@ export const Parent = component$(() => {
 
 export const Child = component$(() => {
   // Get reference to state using MyContext
-  const state = useContext(MyContext);
+  const state = useContext<IMyContext>(MyContext);
```

Comment:
you don't actually need to add it here because it's inferred from the `MyContext` type already

Reply:
Thanks, updated PR.